server/tools/scenery.js

```diff
@@ -87,14 +87,55 @@ function deleteModel(workspace, body) {
     return true;
 }
 
-function createDefinition(workspace, body) {
-    console.log("Create Definition: " + workspace + ": " + json(body));
-    return false;
+function createDefinition(workspace, body) { 
```

Comment:
You'll want to make clean PRs so don't include all the old stuff when you're submitting.